src/containers/List/ListWrapper.tsx

```diff
@@ -140,7 +140,7 @@ const ListWrapper: React.FC = () => {
         filterDefinitions={visibleFilters}
         filterListParameters={{ userId: currentUser?.userId || 0, templateCode: type }}
       />
-      {columns && applicationsRows && (
+      {columns && (
```

Comment:
This was the problem with the "Loading" previously -- the whole "List" wouldn't render until "applicationRows" was defined, by which time it was no longer "Loading". So now we just start with "applicationRows" as an empty array so we still get a List component with a Loading indicator.